test/e2e/top_test.go

```diff
@@ -59,26 +59,32 @@ var _ = Describe("Podman top", func() {
 		Expect(len(result.OutputToStringArray())).To(BeNumerically(">", 1))
 	})
 
-	It("podman top with options", func() {
-		session := podmanTest.Podman([]string{"run", "-d", ALPINE, "top", "-d", "2"})
-		session.WaitWithDefaultTimeout()
-		Expect(session.ExitCode()).To(Equal(0))
-
-		result := podmanTest.Podman([]string{"top", session.OutputToString(), "-o", "pid,fuser,f,comm,label"})
-		result.WaitWithDefaultTimeout()
-		Expect(result.ExitCode()).To(Equal(0))
-		Expect(len(result.OutputToStringArray())).To(BeNumerically(">", 1))
-	})
-
-	It("podman top on container invalid options", func() {
-		top := podmanTest.RunTopContainer("")
-		top.WaitWithDefaultTimeout()
-		Expect(top.ExitCode()).To(Equal(0))
-		cid := top.OutputToString()
-
-		result := podmanTest.Podman([]string{"top", cid, "-o time"})
-		result.WaitWithDefaultTimeout()
-		Expect(result.ExitCode()).To(Equal(125))
-	})
+	//      XXX(ps-issue): for the time being, podman-top and the libpod API
+	//      GetContainerPidInformation(...) will ignore any arguments passed to ps,
+	//      so we have to disable the tests below.  Please refer to
+	//      https://github.com/projectatomic/libpod/pull/939 for more background
+	//      information.
+	//
+	//	It("podman top with options", func() {
```

Comment:
Instead of commenting these out, can you add `Skip("reason")` as the first line of the test?